src/Aspire.Hosting.Azure/Extensions/AzureEventHubsExtensions.cs

```diff
@@ -0,0 +1,101 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Runtime.Versioning;
+using Aspire.Hosting.ApplicationModel;
+using Aspire.Hosting.Azure;
+using Azure.Provisioning;
+using Azure.Provisioning.Authorization;
+using Azure.Provisioning.EventHubs;
+
+namespace Aspire.Hosting;
+
+/// <summary>
+/// Provides extension methods for adding the Azure Service Bus resources to the application model.
+/// </summary>
+public static class AzureEventHubsExtensions
+{
+    /// <summary>
+    /// Adds an Azure Event Hubs Namespace resource to the application model. This resource can be used to create hub resources.
+    /// </summary>
+    /// <param name="builder">The builder for the distributed application.</param>
+    /// <param name="name">The name of the resource.</param>
+    /// <returns></returns>
+    public static IResourceBuilder<AzureEventHubsResource> AddAzureEventHubs(
+        this IDistributedApplicationBuilder builder, string name)
+    {
+#pragma warning disable CA2252 // This API requires opting into preview features
+        return builder.AddAzureEventHubs(name, (_, _, _) => { });
```

Comment:
Change the callback to null since the argument is nullable and you are using `?.Invoke(...)`.